modules/services/imagery_offset.js

```diff
@@ -22,8 +22,95 @@ export default {
         inflight = {};
         offsetCache = rbush();
     },
+    getImageryID: function(url, imageryType) {
+        if (url == null) return;
+        console.log('url=', url)
+        debugger;
+        url = url.toLowerCase();
+        if (imageryType === 'bing' || url.indexOf('tiles.virtualearth.net') > -1)
+            return 'bing';
+        // if (url.indexOf("scanex_irs") > )
+        if (
+            imageryType === 'TMS' &&
+            url.match(/.+tiles\.mapbox\.com\/v[3-9]\/openstreetmap\.map.*/)
```

Comment:
A sed pattern (`s/foo/bar/`) can use any character to delimit, and it's normal to use something other than `/` when dealing with URLs, e.g. `s@foo@bar@`. This avoids excessive escaping. Is this possible with Javascript's regex implementation?